crates/biome_deserialize/README.md

```diff
@@ -183,200 +206,6 @@ assert_eq!(deserialized, None);
 assert_eq!(diagnostics..len(), 1);
 ```
 
-### Deserializing an enumeration of values
```

Comment:
I mostly deleted this because I don't think it makes sense to maintain the code snippets so they stay in sync with the macro. Rust Analyzer offers inspection of macros, so you can easily peek under the hood if you want to.

But indeed I also removed some explanatory text, which I will try to reintroduce in way that makes more sense with the new situation. I left the section for deserializing unions in place, and I just notice there is actually now an easier way to implement that one too (by checking `is_type()` instead of implementing a visitor), so I will update that one and try to retrofit most of the explanatory text there.